cloud/prometheus/prometheus.conf

```diff
@@ -0,0 +1,33 @@
+# Global settings and defaults.
+global {
+  scrape_interval: "10s"
+  evaluation_interval: "10s"
```

Comment:
Might be a bit hot for long-term production use, but kinda handy for now while it's barely working :)
We can certainly crank it down later on once it's all settled in.